content/docs/guide.md

```diff
@@ -72,6 +72,23 @@ i.e. the same you'd get by pip installing from Github.
     $ packit
     Usage: packit [OPTIONS] COMMAND [ARGS]...
 
+Based upon the command you need to performer you should configure your container with the needed 
+secrets and configuration files like in the following examples:
+
+#### packit copr-build
+
+    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash
```

Comment:
I think you can use the `:z`/`:Z` option for volume instead of `--security-opt label=disable`.

```suggestion
    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src quay.io/packit/packit bash
```

Reply:
Yes the :z option on mounted volumes is working. Since I copied it from the documentation, do you want me to update also the other examples?

Reply:
Nice! I see you have already updated that.